content/en/docs/tutorials/trust-policy.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "Create a trust policy and verify artifacts"
+description: "How to create a trust policy and use it to verify artifacts"
+type: docs
+weight: 1
+---
+
+
+quick details on trust policy
+link to concepts doc
+
+To verify the container image, configure the trust policy to specify trusted identities that sign the artifacts, and level of signature verification to use. For more details, see [trust policy spec]({{< ref "/docs/concepts/trust-store-trust-policy-specification#trust-policy" >}}).
+
+## Create an example registry with an image
+
+**WARNING:** The following example creates a registry with [oras-project/registry](https://github.com/oras-project/distribution/pkgs/container/registry). This registry should only be used for development purposes. When using other registries, ensure the registry is compatible with OCI Image specification v1.1.0. Starting with `v1.0.0-rc.1` of `notation`, by default, signatures are stored using [OCI Artifact Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md), which is defined in [OCI Image spec v1.1.0](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc2)).
+
+```console
+docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v1.0.0-rc.3
+```
+
+**NOTE:** For Apple silicon, add the `--platform linux/arm64` parameter.
+
+Use `docker buildx build` and `docker push` to build and push a sample image to your registry.
+
+```console
+docker buildx build -t localhost:5000/net-monitor:v1 https://github.com/wabbit-networks/net-monitor.git#main
+docker push localhost:5000/net-monitor:v1
+```
+
+Save the digest value of the image from the output of `docker push`. For example:
+
+```output
+The push refers to repository [localhost:5000/net-monitor]
+2556c54bfdf3: Pushed
+fb6ca4f9c8d3: Pushed
+ded7a220bb05: Pushed
+v1: digest: sha256:1111111111111111111111111111111111111111111111111111111111111111 size: 942
+```
+
+In the above example, the digest value of the image is *sha256:1111111111111111111111111111111111111111111111111111111111111111*.
+
+## Create example certificates and sign the image
+
+Use `notation cert generate-test` to generate three example certificates and trust stores for signing the image.
+
+```console
+notation cert generate-test valid-example --default
```

Comment:
Is it a good idea to explain what the name indicates "valid-example", "expired-example" and "missing-example", so that user can get some clues before the different scenarios explained below.

For example, "valid-example" means the signature is valid, and certificates are correct. "expired-example" means the signature is expired. "missing-example" means the certificates are missing or wrong.

Reply:
good call, fixed